config/jobs/kubernetes-sigs/sig-windows/release-1.22-windows-presubmits.yaml

```diff
@@ -0,0 +1,172 @@
+# generated by ./config/jobs/kubernetes-sigs/sig-windows/generate-presubmits.sh.
+presubmits:
```

Comment:
Should we add these presubmits to https://testgrid.k8s.io/sig-windows-presubmit?

Reply:
It looks like we only do the jobs against the main branch. I'd say we leave it that way for now